Frontend/proxy-conf.json

```diff
@@ -0,0 +1,15 @@
+{
+    "/api/*": {
+    
+    "target": "http://localhost:3000/api",
```

Comment:
@brownm26csp I think we can run the API on a different port, like 3001. This way, we can just hit that endpoint when connecting the front end and back end. Then I think we can move these variables to an env file, I think this would be simpler and then we can have different files for each env ie env.production and env.development